scripts/sync_data.py

```diff
@@ -59,6 +64,17 @@ def download_contrarianmba_airtable_records():
         f.write("\n")
 
 
+def download_image_urls():
+    with open(CONTRARIANMBA_RAW_JSON_FILENAME, "r") as f:
+        data = json.loads(f.read())
+        books = [Book.from_raw(record) for record in data["books"]]
```

Comment:
Since this function is below `process_records`, go ahead and use the processed JSON file, not the raw JSON.